templates/csharp/workflow/Commands/GenericCommandHandler.cs.tpl

```diff
@@ -0,0 +1,58 @@
+using Microsoft.Bot.Builder;
+using Microsoft.TeamsFx.Conversation;
+
+namespace {{SafeProjectName}}.Commands
+{
+    /// <summary>
+    /// The <see cref="GenericCommandHandler"/> registers patterns with the <see cref="ITeamsCommandHandler"/> and
+    /// responds with appropriate messages if the user types general command inputs, such as "hi", "hello", and "help".
+    /// </summary>
+    public class GenericCommandHandler : ITeamsCommandHandler
+    {
+        private readonly ILogger<GenericCommandHandler> _logger;
+    
+        public IEnumerable<ITriggerPattern> TriggerPatterns => new List<ITriggerPattern>
+        {
+            // Used to trigger the command handler if the command text contains 'helloWorld'
```

Comment:
Need to update the code comment here to consistent with the RegExp.

Reply:
Thanks, updated.